src/items.md

```diff
@@ -17,8 +17,8 @@
 > &nbsp;&nbsp; &nbsp;&nbsp; | [_Trait_]\
 > &nbsp;&nbsp; &nbsp;&nbsp; | [_Implementation_]\
 > &nbsp;&nbsp; &nbsp;&nbsp; | [_ExternBlock_]\
-> &nbsp;&nbsp; &nbsp;&nbsp; | _Macro_\
-> &nbsp;&nbsp; &nbsp;&nbsp; | _MacroDefinition_\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_MacroInvocationSemi_]&nbsp;[^novis]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_MacroRulesDefinition_]&nbsp;[^novis]\
```

Comment:
Was this deliberately not written like this?
_OuterAttribute_ ( _Visibility_ <sup>?</sup> ( _Module_ | ... | _ExternBlock_ ) | _MacroInvocationSemi_ | _MacroRulesDefinition_ )

Reply:
I was trying to avoid making it too nested.  In that example, there would need to be another set of parenthesis, like: _OuterAttribute_<sup>\*</sup> ( ( _Visibility_<sup>?</sup> ( _Module_ | ... | _ExternBlock_ ) ) | _MacroInvocationSemi_ | _MacroRulesDefinition_ ) which seems a little unwieldy.  I pushed a variation that removes the footnote, let me know if it looks any better.

Reply:
I think I prefer this version because the macro can expand to an item with a visibility.